apps/api/src/main/java/dk/treecreate/api/newsletter/Newsletter.java

```diff
@@ -0,0 +1,103 @@
+package dk.treecreate.api.newsletter;
+
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "Newsletter",
```

Comment:
It seems that this is the email of a subscriber, and not a newsletter itself. I think its best if we name it "emailSubscriber" og just "subscriber". I feel confused when creating a newsletter, and it being a subscriber. Or have i misunderstood the function of this class?

Reply:
Since its called newsletter in the next files as well, i can overlook this and just get used to it, if you disagree or its too much effort to change. :)